common/processor/class-model/src/main/java/module-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The class model generator.
+ */
+module io.helidon.common.processor.model {
```

Comment:
inconsistent directory with module name, artifactId, and package name.
If `processor/class-model`, then it should be `io.helidon.common.processor.classmodel` and `helidon-common-processor-class-model` and package `io.helidon.common.processor.classmodel`

Reply:
fixed